pkgs/top-level/python-packages.nix

```diff
@@ -3392,6 +3392,24 @@ let
     };
   };
 
+  gateone = buildPythonPackage rec {
+    name = "gateone-1.2-0d57c3";
+    src = pkgs.fetchFromGitHub {
+      rev = "11ed97c663b3e8c1b8eba473b5cf8362b10d57c3";
+      owner= "liftoff";
+      repo = "GateOne";
+      sha256 ="0zp9vfs6sqbx4d0g45kkjinfmsl9zqwa6bhp3xd81wx3ph9yr1hq";
+    };
+    buildInputs = with pkgs.pythonPackages; [tornado futures html5lib readline];
+    propagatedBuildInputs = with pkgs.pythonPackages; [tornado futures html5lib readline pkgs.openssl];
+    meta = {
+      homepage = https://liftoffsoftware.com/;
+      description = "GateOne is a featured terminal emulator";
```

Comment:
I've never heard "featured" used like that, and it *is* far more than a terminal emulator. [What about](http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes): `"A web based terminal emulator and SSH client"`?